packages/cambodia/cambodia.svg

```diff
@@ -0,0 +1,109 @@
+<svg
+   xmlns="http://www.w3.org/2000/svg"
+   viewBox="0 0 655 655"
+   aria-label="Map of Cambodia"
+>
+  <path
+     id="banteay-meanchey"
+     title="Banteay Meanchey"
```

Comment:
Could you use `name` instead of `title` attributes please?

Reply:
Yes, I had renamed it.